uint8 scenarioIdx
bool active
---
bool success